src/sentry/replays/endpoints/project_replay_summary.py

```diff
@@ -158,6 +158,17 @@ def post(self, request: Request, project: Project, replay_id: str) -> Response:
         # Download segment data.
         # XXX: For now this is capped to 100 and blocking. DD shows no replays with >25 segments, but we should still stress test and figure out how to deal with large replays.
         segment_md = fetch_segments_metadata(project.id, replay_id, 0, MAX_SEGMENTS_TO_SUMMARIZE)
+        if len(segment_md) >= MAX_SEGMENTS_TO_SUMMARIZE:
+            logger.warning(
+                "Replay hit max segment limit.",
```

Comment:
I would add that this is for summaries specifically and maybe what the max limit is

```suggestion
                "Replay Summary: Max segment limit (100)",
```

Reply:
Related - just added a seer request size log too! (log when >100k char)